src/components/button/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import style from './button.module.scss';
+
+export const ButtonTypes = {
```

Comment:
Would it be easier to just export two buttons rather than making the parent component import both the style and the button and then applying the exported style to the button?